S3/S3.py

```diff
@@ -1064,6 +1065,27 @@ def set_acl(self, uri, acl):
         response = self.send_request(request)
         return response
 
+    def set_versioning(self, uri, status):
+        headers = SortedDict(ignore_case = True)
+        headers['content-type'] = 'application/xml'
```

Comment:
I don't think that you should set that.
We don't set it usually and it is not an issue. better to stay consistent.